packages/block-scripts/package.json

```diff
@@ -48,13 +48,19 @@
     "typescript": "4.8.2",
     "typescript-json-schema": "^0.53.0",
     "uuid": "^8.3.2",
-    "webpack": "^5.72.0",
+    "webpack": "^5.74.0",
```

Comment:
I added `@types/webpack-assets-manifest` and `@types/webpack-bundle-analyzer` to fix missing typings and TSC reported type mismatches. Upgrading webpack by two minor versions fixed this (webpack comes with built-in typings).